Common/Data/Market/OptionContract.cs

```diff
@@ -26,6 +26,7 @@ namespace QuantConnect.Data.Market
     public class OptionContract
     {
         private Lazy<OptionPriceModelResult> _optionPriceModelResult = new(() => OptionPriceModelResult.None);
+        private readonly decimal _strikeMultipler;
```

Comment:
It doesn't seem we need to keep the private strike multiplier? Let's remove it if possible 👍